components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +157,548 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
     {
       "effect": "ALLOW",
       "actions": [
-        "iam:users:list"
-      ]
+        "iam:users:list",
+        "iam:users:get"
+      ],
+      "projects": [
+        "*"
+      ],
     },
     {
       "effect": "ALLOW",
       "role": "editor"
     }
+  ],
+  "projects": [
+    "admin-group-1",
+    "it-support",
   ]
 }
 ```
 
-Assuming you stored the above JSON in the file "policy.json", pass it to `curl` to create the policy:
+### Listing Policies
 
-```bash
-curl -s -H "api-token: $TOKEN" -d @policy.json https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
-```
+The output includes both *Chef-managed* and *Custom* policies.
+Policies migrated from IAM v1 are included as *Custom* policies.
+Policy names that begin with `[Legacy]` are default policies migrated from v1.
+We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
```

Comment:
could we add something that says we recommend deleting them after moving associated members to the new chef-managed policies?